h/views/activity.py

```diff
@@ -573,3 +575,23 @@ def _copy_params(request, params=None):
         del params['q']
 
     return params
+
+def _record_search_api_usage_metrics(params, record_param=newrelic.agent.add_custom_parameter):
```

Comment:
We should change this function name since it's not actually recording api metrics. Maybe `_record_search_query_param_usage`.